cmake/modules/hunter_experimental_add_host_project.cmake

```diff
@@ -0,0 +1,66 @@
+# Copyright (c) 2016 Fabian Winnen
+# All rights reserved.
+#
+# Warning:
+#   This module is in experimental stage.
+#   TODO: more disclamers
+#   TODO: notice that we clear some env variables?
+#
+# Adds a project with CMakeList.txt as seperate build with default (host) toolchain
+#
+# Usage:
+#   hunter_experimental_add_host_project(host)
+#
+# sets HUNTER_HOST_ROOT variable to point at host project's binary directory
+message(WARNING "You are using a hunter module in experimental stage:\nhunter_experimental_add_host_project\n")
```

Comment:
Let's put it into function, not when user do include this module.

Also I was thinking about more scary message like:

```
*****************************************************************
                                 !!! WARNING !!!

You're using experimental feature. This may not work well
and can be removed anytime. It's a workaround for existsing
feature request:

    https://github.com/ruslo/hunter/issues/495

*****************************************************************
```